### DISCUSSION TITLE
deleteObject() S3

### DISCUSSION THREAD

**Discussion by ravinemala-jp** [aws/aws-sdk-js#4154] [Q&A]
I am trying to delete and object from S3 , it deletes an object but does not return any exception or error when object not found.
always returning success code 200. please help how to handle exception when object not found. any example will be appreciated.

**Comment by ghost** [comment]
I'm also having issues when deleting an object from S3.... the object is indeed deleted and a 204 is returned but I also get a 403 forbidden  with the following error:

"err:  SignatureDoesNotMatch: The request signature we calculated does not match the signature you provided. Check your key and signing method."
![Screenshot from 2022-07-22 13-19-55](https://user-images.githubusercontent.com/100789949/180500799-59f7476e-039d-49a2-8664-c2c34786c9c4.png)
